persistence-typed/src/main/scala/org/apache/pekko/persistence/typed/internal/Running.scala

```diff
@@ -751,7 +751,7 @@ private[pekko] object Running {
     }
 
     def onGetSeenSequenceNr(get: GetSeenSequenceNr): PersistingEvents = {
-      get.replyTo ! state.seenPerReplica(get.replica)
+      get.replyTo ! state.seenPerReplica.getOrElse(get.replica, 0L)
```

Comment:
I think we should extract the 0L to a constant if possible.

Reply:
the compiler will handle this nicely - I don't think we need a constant for 0